compiler/onecc-docker/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:18.04 
+
+RUN apt-get update 
```

Comment:
When writing a `Dockerfile`, we have to minimize the number of layers. The `RUN`, `COPY`, `ADD` create layers. If you execute only one command at one layer, each execution result is saved as the cache. In this case, if you execute `apt-get update`, all updated files will create a cache. Rather than implementing it like this, it is more efficient to implement it by executing `apt-get update` on one `RUN` command, installing the package you actually want to install, and removing files that have been used unnecessarily.

related guideline : https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run